indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/AbstractMultiPhaseParallelIndexingTest.java

```diff
@@ -176,7 +176,7 @@ void runTaskAndVerifyStatus(Task task, TaskState expectedTaskStatus)
   {
     task.addToContext(Tasks.FORCE_TIME_CHUNK_LOCK_KEY, lockGranularity == LockGranularity.TIME_CHUNK);
     TaskStatus taskStatus = getIndexingServiceClient().runAndWait(task);
-    Assert.assertEquals(expectedTaskStatus, taskStatus.getStatusCode());
+    Assert.assertEquals(taskStatus.toString(), expectedTaskStatus, taskStatus.getStatusCode());
```

Comment:
Maybe a longer message like: "Expected task status to be ..."

Reply:
Sure. this is the actual task status though.